guides/RELEASE-PROCESS.md

```diff
@@ -56,6 +53,8 @@
       ```
       This will run `npm publish` for each public package in `packages/`.
       **Note**: You should only publish the `master` branch to NPM. The publish script above will check out the `master` branch if it isn't currently the `HEAD`.
+   
+   1. Publish the release notes after the npm package is updated
```

Comment:
@line47 I don't know if this is too complicated, but maybe these backstop changes can be included in the same PR/branch as the one  after the prepublish script. This way each release would only have 1 PR that would do 2 things, bump the package number and update the backstop reference files? The downside is that the branch has to be left open until everything is deployed...

Reply:
Yeah that makes sense :)